schema/policy_test.go

```diff
@@ -525,6 +525,43 @@ var _ = Describe("Policies", func() {
 			})
 		})
 
+		Describe("Built-in policies in gohan.json", func() {
+			var adminAuth, domainAdminAuth, memberAuth Authorization
+
+			BeforeEach(func() {
+				ClearManager()
+				manager = GetManager()
+				Expect(manager.LoadSchemaFromFile("embed://etc/schema/gohan.json")).To(Succeed())
+
+				adminAuth = NewAuthorizationBuilder().
+					WithRoleIDs("admin").
+					WithTenant(Tenant{ID: adminTenantID, Name: "admin"}).
+					BuildAdmin()
+				domainAdminAuth = NewAuthorizationBuilder().
+					WithRoleIDs("admin").
+					WithDomain(Domain{ID: "domainID", Name: "domainName"}).
+					BuildScopedToDomain()
+				memberAuth = NewAuthorizationBuilder().
+					WithRoleIDs("Member").
+					WithTenant(Tenant{ID: demoTenantID, Name: "demo"}).
+					BuildScopedToTenant()
+			})
+
+			It("should allow operations for admin-scoped token", func() {
+				p, _ := manager.PolicyValidate("read", "/v2.0/networks", adminAuth)
```

Comment:
Is `/v2.0/networks` somehow meaningful in this test? If not, I'd go for something more descriptive, like `/any/path/without/a/custom/policy`